guides/intergration/oidc.md

```diff
@@ -0,0 +1,120 @@
+# OpenID Connect
+
+Trento integrates with identity providers that use the OpenID Connect **(OIDC)** protocol to authenticate users accessing the console. Authorization for specific abilities/permissions is managed by Trento, which means that only basic user information is retrieved from the external identity provider **(IDP)**.
+
+## Enabling OIDC
+
+The OIDC authentication is **disabled by default**.
+
+Provide the following environment variable to enable OIDC feature when starting Trento.
+
+```
+ENABLE_OIDC=true
+OIDC_CLIENT_ID=<<OIDC_CLIENT_ID>>
+OIDC_CLIENT_SECRET=<<OIDC_CLIENT_SECRET>>
+OIDC_BASE_URL=<<OIDC_BASE_URL>>
+OIDC_CALLBACK_URL=<<OIDC_CALLBACK_URL>>
```

Comment:
Let's put a comment in the `OIDC_CALLBACK_URL`, because this field is optional, and rarely used.
I would put some test in between the 1st mandatory options, and the field after that